doc/documentation.rst

```diff
@@ -1,41 +1,41 @@
 .. _documentation:
 
 ************************
-Generating documentation
+Generating Documentation
 ************************
 
 Prerequisites
 =============
 
-Documentation in dune is done courtesy of the odoc_ tool. Therefore, to
-generate documentation in dune, you will need to install this tool. This
-should likely be done with opam:
+Documentation in Dune is done courtesy of the ``odoc_`` tool. Therefore, to
```

Comment:
Same question. It would be nice to verify that the link to the odoc project is maintained.

Reply:
Thanks for pointing this out. It should be `odoc` without the extra space, although I would prefer Odoc or ODoc, it's not my call. 😺

Reply:
OH! Now I see what you're saying about the link. Apologies for the misunderstanding.